File: 1. Begginer/10- Pida al usuario dos números reales (que se guardarán en las variables dato1 y dato2).Si dato2 es cero,deberá mostrar un mensaje de error, y en caso contrario mostrará en pantalla el resultad/main.cpp
#include <iostream>

using namespace std; // accesing to class to not use in code std::method

int main(void) {
	float data1;
	float data2;
	cout << "Ingrese un número real: " << endl;
	cin >> data1;
	cout << "Ingrese otro número real: " << endl;
	cin >> data2;
	if (data2 == 0) {
		cout << "Error: No se pueden dividir números entre 0";
	} else {
		cout << "El resultado de dividir " << data1 << " entre " << data2 << " es igual a: " << data1/data2;
	}
}

File: 1. Begginer/10- Pida al usuario dos números reales (que se guardarán en las variables dato1 y dato2).Si dato2 es cero,deberá mostrar un mensaje de error, y en caso contrario mostrará en pantalla el resultad/main.go
package main
import "fmt"

func main() {
	var data1 float32
	var data2 float32
	fmt.Println("Ingrese un número real: ")
	fmt.Scanf("%f", &data1)
	fmt.Println("Ingrese otro número real: ")
	fmt.Scanf("%f", &data2)
	if data2 == 0 {
		fmt.Println("Error: No se pueden dividir números entre 0")
	} else {
		/* Un float32 o 64 en golang se representa en números binarios con 32 bits o 64 bits */
		fmt.Printf("El resultado de dividir %f entre %f es igual a: %f", data1, data2, data1/data2)
		/* Printf funciona igual que printf de C */
	}
}